packages/@guardian/source-package-validation/README.md

```diff
@@ -0,0 +1,23 @@
+# Source Package Validation
+
+This project is used to verify the output of the Source packages.
+
+## How?
+
+It runs two checks:
+
+1. Verify that no exports have changed in the published packages
+
+    This works similar to a snapshot test in that, if anything does change, the test must be updated. This ensures that subsequent releases haven't broken the package structure or unexpectedly removed exports from the public API. It uses jest to run the test.
+
+2. Verify that no types have been removed
+
+    This test works by importing all of the types from all packages into a file and then running `tsc`. It will capture types that have been removed but not fail when new types are added so, for now, the onus is on contributors and reviewers to remember to add new types to this file.
```

Comment:
```suggestion
    This test works by importing all of the types from all packages into a file and then running `tsc`. It will capture types that have been removed but not fail when new types are added so, for now, the onus is on contributors and reviewers to **remember to add new types to this file**.
```